src/reducers/reducers.test.js

```diff
@@ -301,6 +302,17 @@ describe('Reducer', () => {
     });
   });
 
+  describe('TOGGLE_CODE', () => {
+    it('should toggle the value of hoveredParameters', () => {
```

Comment:
The description here is irrelevant to the test - it should be something along the lines as `should toggle / update the value of `visible.code` in the state`